src/sentry/coreapi.py

```diff
@@ -41,7 +41,9 @@
 from sentry.utils import json
 from sentry.utils.auth import parse_auth_header
 from sentry.utils.csp import is_valid_csp_report
-from sentry.utils.http import is_valid_ip, origin_from_request
+from sentry.utils.http import (origin_from_request,
```

Comment:
Is this what our automatic code formatter is spitting out now? Otherwise, this is a very weird formatting style. If it's from the tool, it's fine.